CONTRIBUTING.md

```diff
@@ -2,9 +2,11 @@
 
 1. Fork it!
 2. Create your feature branch: `git checkout -b my-new-feature`
-3. Commit your changes: `git commit -am 'Add some feature'`
-4. Push to the branch: `git push origin my-new-feature`
-5. Submit a pull request :D
+3. Test it: `cargo test`
+4. Lint it: `cargo +nightly clippy --all-targets -- -D warnings`
+5. Commit your changes: `git commit -am 'Add some feature'`
+6. Push to the branch: `git push origin my-new-feature`
+7. Submit a pull request :D
```

Comment:
We could advocate for github cli.

`gh pr create`

Reply:
Please do suggest a PR for that, I haven't looked into accessibility for it across platforms etc.